client/jetpack-cloud/sections/sidebar-navigation/manage-selected-site.tsx

```diff
@@ -1,8 +1,186 @@
-// selector. It'll display menu options like Activity Log, Backup, Social, etc.
-// FIXME: Add menu items and the right path
-const ManageSelectedSiteSidebar = () => <NewSidebar path="/" menuItems={ [] } />;
+const useMenuItems = ( {
+	siteId,
+	path,
+	isAgency,
+}: {
+	siteId: number | null;
+	path: string;
+	isAgency: boolean;
+} ) => {
+	const translate = useTranslate();
+	const siteSlug = useSelector( getSelectedSiteSlug );
+	const hasBackups = useSelector( ( state ) =>
+		siteHasFeature( state, siteId, WPCOM_FEATURES_BACKUPS )
+	);
+	const hasScan = useSelector( ( state ) => siteHasFeature( state, siteId, WPCOM_FEATURES_SCAN ) );
+
+	const isAdmin = useSelector( ( state ) => canCurrentUser( state, siteId, 'manage_options' ) );
+
+	const isWPCOM = useSelector( ( state ) => getIsSiteWPCOM( state, siteId ) );
+
+	const isWPForTeamsSite = useSelector( ( state ) => isSiteWPForTeams( state, siteId ) );
+
+	const shouldShowSettings =
+		useSelector( ( state ) => canCurrentUser( state, siteId, 'manage_options' ) ) &&
+		( hasBackups || hasScan );
+
+	const shouldShowPurchases =
+		useSelector( ( state ) => canCurrentUser( state, siteId, 'own_site' ) ) &&
+		isSectionNameEnabled( 'site-purchases' );
+
+	const isPluginManagementEnabled = config.isEnabled( 'jetpack/plugin-management' );
+
+	return useMemo(
+		() =>
+			[
+				{
+					icon: <JetpackIcons icon="activity-log" size={ 24 } />,
+					path: '/',
+					link: `/activity-log/${ siteSlug }`,
+					title: translate( 'Activity Log' ),
+					onClickMenuItem: redirectPage,
+					trackEventName: 'calypso_jetpack_sidebar_activity_clicked',
+					enabled: isAdmin,
+					isSelected: itemLinkMatches( path, `/activity-log/${ siteSlug }` ),
+				},
+				{
+					icon: plugins,
+					path: '/',
+					link: pluginsPath( siteSlug ),
+					title: translate( 'Plugins' ),
+					onClickMenuItem: redirectPage,
+					trackEventName: 'calypso_jetpack_sidebar_plugins_clicked',
+					enabled: isPluginManagementEnabled && isAgency,
+					isSelected: itemLinkMatches( path, pluginsPath( siteSlug ) ),
+				},
+				{
+					icon: cloud,
+					path: '/',
+					link: backupPath( siteSlug ),
+					title: translate( 'Backup' ),
+					onClickMenuItem: redirectPage,
+					trackEventName: 'calypso_jetpack_sidebar_backup_clicked',
+					enabled: isAdmin && ! isWPForTeamsSite,
+					isSelected: itemLinkMatches( path, backupPath( siteSlug ) ),
+				},
+				{
+					icon: shield,
+					path: '/',
+					link: scanPath( siteSlug ),
+					title: translate( 'Scan' ),
+					onClickMenuItem: redirectPage,
+					trackEventName: 'calypso_jetpack_sidebar_scan_clicked',
+					enabled: isAdmin && ! isWPCOM && ! isWPForTeamsSite,
+					isSelected: itemLinkMatches( path, scanPath( siteSlug ) ),
+				},
+				{
+					icon: search,
+					path: '/',
+					link: `/jetpack-search/${ siteSlug }`,
+					title: translate( 'Search' ),
+					onClickMenuItem: redirectPage,
+					trackEventName: 'calypso_jetpack_sidebar_search_clicked',
+					enabled: isAdmin,
```

Comment:
Looks like this is always enabled currently.

Reply:
I am not 100% sure if this is always the case. I haven't done any research about this flag, and I just copied the existing logic to be on the safe side.

Reply:
I looked at this file: `client/components/jetpack/sidebar/menu-items/index.jsx`, and it seems like it is always enabled. Not sure if I'm missing something here. 

<img width="740" alt="Screenshot 2023-10-19 at 3 43 10 PM" src="https://github.com/Automattic/wp-calypso/assets/10586875/b283b083-8ca6-431c-b0da-2518597cca36">

Reply:
Aha. I didn't get what you meant at first, but yes, you are right this is always true. Thanks for pointing this out 👍